lib/lexisnexis/request.rb

```diff
@@ -22,6 +23,7 @@ def initialize(attributes)
     def send(response_options: {})
       conn = Faraday.new do |f|
         f.options[:timeout] = timeout
+        f.basic_auth config.username, config.password
```

Comment:
I removed the manual Base64-ing  in `encoded_credentials` so we could use this nice built-in helper. But I did update `spec/support/shared_examples/request.rb` to make sure the header is still there